### ISSUE TITLE
dev-requirements for python

### ISSUE THREAD

**Issue by david-bielen** [dbousque/batch_jaro_winkler#8]
Can you review the current requirements for python (https://github.com/dbousque/batch_jaro_winkler/blob/master/python/dev-requirements.txt)?
I succesfully installed the package with cython = 0.29.33 (with python 3.10.4).

**Comment by dbousque** [comment]
Thanks for your message. Was the `pip` install command not working for you? Why did you build this package manually?

**Comment by david-bielen** [comment]
pip fails with python3.11
```

python3.11 -m pip install batch-jaro-winkler
Collecting batch-jaro-winkler
  Using cached batch_jaro_winkler-0.1.1.tar.gz (85 kB)
  Preparing metadata (setup.py) ... done
Installing collected packages: batch-jaro-winkler
  DEPRECATION: batch-jaro-winkler is being installed using the legacy 'setup.py install' method, because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. pip 23.1 will enforce this behaviour change. A possible replacement is to enable the '--use-pep517' option. Discussion can be found at https://github.com/pypa/pip/issues/8559
  Running setup.py install for batch-jaro-winkler ... error
  error: subprocess-exited-with-error

  × Running setup.py install for batch-jaro-winkler did not run successfully.

cbatch_jaro_winkler.c:216:12: fatal error: 'longintrepr.h' file not found
```

**Comment by dbousque** [comment]
I released a fix in version 0.1.2. Thanks for the report!
